test/e2e/standard/test.js

```diff
@@ -17,24 +17,192 @@ const serverUrl = 'http://localhost:8383';
 const userEmail = 'x@example.com';
 const userPassword = 'secret1234';
 
+describe('Cache headers', () => {
+  const undici = require('undici');
```

Comment:
nit: maybe move this to the top of the file with other requires.

Reply:
I've included it here to keep it`undici` scoped to these specific tests.  If it's preferable, I can refactor these tests into a separate file, but I'd rather do that in a follow-up PR.

Reply:
scoped `undici` is fine.